pkgs/json_syntax_generator/lib/src/parser/schema_analyzer.dart

```diff
@@ -44,11 +44,15 @@ class SchemaAnalyzer {
   /// Generate public setters for these class names.
   final List<String> publicSetters;
 
+  /// For subtypes of these classes, the union tag values are exposed.
+  final List<String> visbleUnionTagValues;
```

Comment:
I don't understand what the word `visbleUnionTagValues` means - could you rename this or add some documentation?